c.md

```diff
@@ -471,4 +471,9 @@ All functions defined under `<stdlib.h>` header file. These randomized functions
 
 * `gcc prog.c` : compiles prog.c into a.out, run result with  `./a.out`.
 * `gcc -o prog prog.c` : compiles prog.c into prog, run result with `./prog`.
-* `gcc -g -o prog prog.c` : as above, but allows for debugging.
\ No newline at end of file
+* `gcc -g -o prog prog.c` : as above, but allows for debugging.
+
+##use of fflush()
```

Comment:
Add space after `##` to fix markdown formatting

Reply:
Added space after ##.Can you check now..